cmake/projects/OpenSSL/externals/nasm_path_configure.cmake.in

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.0)
+
+message("################################")
+set(ENV{PATH} "$ENV{NASM_ROOT}/bin;$ENV{PATH}")
```

Comment:
Use `@NASM_ROOT@` here.

Reply:
yes.. I just pushed. Now path is correct but still "NASM not found"